### PR TITLE
corrected sentence

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #7199 , so i corrected sentence

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/presence-admins.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
 || ![Purple circle with arrow, indicates Out of office](media/Presence_OOF.png) Out of Office. Out of Office is used when an automatic reply is set. (Available in Outlook only.) |
 |||
  > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:49]:
```suggestion
 > [!NOTE]
```

File: Teams/presence-admins.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
...
- > For users that there mailbox is hosted onprem it is expected to have presence delays with a maximun of an hour.
+ > For users that their mailbox is hosted on-premises, it is expected to have presence delays with a maximum of an hour.
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:50]:
```suggestion
 > For users whose mailboxes are hosted on-premises, it is expected to have presence delays of a maximum of an hour.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RAJU2529 - thank you for the correction. Just a further suggestion to improve readability.

File: Teams/presence-admins.md [hunk 3]
```diff
@@ -46,8 +46,8 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
 || ![Open gray circle, indicates status unknown](media/Presence_Unknown.png) Status unknown|
 || ![Purple circle with arrow, indicates Out of office](media/Presence_OOF.png) Out of Office. Out of Office is used when an automatic reply is set. (Available in Outlook only.) |
 |||
- > [!Note]
- > For users that there mailbox is hosted onprem it is expected to have presence delays with a maximun of an hour.
+ > [!NOTE]
+ > For users whose mailboxes are hosted on-premises, it is expected to have presence delays of a maximum of an hour.
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:50]:
```suggestion
 > For users whose mailboxes are hosted on-premises, it is expected to have presence delays of a maximum of an hour.
 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RAJU2529 Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**